### PR TITLE
Update CLA Message and remove CLA munger

### LINKED ISSUE #2962: Add CNCF contact information to k8s-ci-robot CLA message
Exactly this text.

```
Please email the CNCF helpdesk at helpdesk@rt.linuxfoundation.org for support with the CLA.
```

Would also be nice to add that text to the FAQ https://github.com/kubernetes/kubernetes/wiki/CLA-FAQ.

ref. kubernetes/kubernetes#42409
cc @cblecker

### PR DESCRIPTION
**Pull request by cblecker**
This PR removes the CLA plugin from the munger (appears to not be used), and also updates the CLA message to include info on contacting the CNCF helpdesk.

Fixes #2962.

### CODE REVIEW

File: prow/plugins/cla/cla.go [hunk 1]
```diff
@@ -42,6 +42,7 @@ It may take a couple minutes for the CLA signature to be fully registered; after
 
 - If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address.  Check your existing CLA data and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
 - If you signed the CLA as a corporation, please sign in with your organization's credentials at <https://identity.linuxfoundation.org/projects/cncf> to be authorized.
+- If you have done the above and are still having issues with the CLA being reported as unsigned, please open a ticket with the CNCF helpdesk: helpdesk@rt.linuxfoundation.org
```
Comment (rmmh) [inline prow/plugins/cla/cla.go]: Match the issue text: `s/open a ticket with/email/`

**Comment by rmmh** [comment]
This looks fine apart from the ambiguity of "open a ticket".

/lgtm

**Comment by cblecker** [comment]
@rmmh comments addressed

**Comment by spiffxp** [comment]
@cblecker needs rebase

**Comment by cblecker** [comment]
@spiffxp @rmmh rebased

**Comment by spiffxp** [comment]
@cblecker sorry, I was too quick on the merge button there, there are some configmaps/flags that should be removed as well, can you followup with a PR that addresses these?
- https://github.com/kubernetes/test-infra/blob/master/mungegithub/submit-queue/deployment.yaml#L53
- https://github.com/kubernetes/test-infra/blob/master/mungegithub/submit-queue/deployment.yaml#L254-L258
- https://github.com/kubernetes/test-infra/blob/master/mungegithub/submit-queue/deployment/contrib/configmap.yaml#L51
- https://github.com/kubernetes/test-infra/blob/master/mungegithub/submit-queue/deployment/kubernetes/configmap.yaml#L135

**Comment by cblecker** [comment]
@spiffxp on it

**Comment by cblecker** [comment]
@spiffxp actually, looking at those, I think those are looking for the github context to ensure that the SQ can merge. I think those have to stay

**Comment by spiffxp** [comment]
@cblecker I think that's what https://github.com/kubernetes/test-infra/blob/master/mungegithub/submit-queue/deployment/kubernetes/configmap.yaml#L30 is for

**Pull request merged**